doc/nrf/releases_and_maturity/migration/migration_guide_2.9.rst

```diff
@@ -32,7 +30,6 @@ nRF54H20
 ========
 
 This section describes the changes specific to the nRF54H20 SoC and DK support in the |NCS|.
-For more information on changes related to samples and applications usage on the nRF54H20 DK, see :ref:`migration_2.9_required_nrf54h`.
 
 DK compatibility
```

Comment:
This entire section for the nRF54H20 should be removed. It is not really part of this release, @peknis